pkg/config/config.go

```diff
@@ -73,6 +73,11 @@ var (
 			NoPrefix:   true,
 			Required:   true,
 		},
+		{
+			Name:     "exclude_nodes_by_label",
+			Help:     "jenkins exclude nodes by label",
```

Comment:
```suggestion
			Help:     "exclude jenkins nodes by label",
```

Reply:
`node == agent`? not `node == cloud_instance`?